pkg/ca-certificates/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM linuxkit/alpine:f0169b60fb260d74025496ae6fd93213fecaba8f as alpine
+FROM linuxkit/alpine:c5021a113803f7608e3908014d316c3490183d0b@sha256:4c7a80173c71ca5019dc56f40442b3a8345f141dd46593bd1fe6d130294d07cb as alpine
```

Comment:
The plan was not to have sha256 hashes, as we are going to use notary so tags are fine.